docs/running-node-aws-terraform-ansible/deployment.md

```diff
@@ -0,0 +1,236 @@
+        2
+    ]
+}
+```
+
+- Requires `Private Sentries` being enabled
+- When `root_domain_name` parameter is set, will be serving requests under `on.<root_domain_name>` domain
+    - Name servers for `root_domain_name` servers must be pointed to AWS and managed by the AWS account
+- TLS can be enabled or disabled
+- Can be configured to run on multiple regions
+
+#### Prometheus (optional)
+```hcl
+prometheus_config = {
+  enable        = true
+  instance_type = "t3.small"
+}
+```
+
+- Requires `Private Sentries` being enabled
+- When enabled runs a dedicated Prometheus server on Private Sentries VPC to collect Tendermint metrics from all DCL nodes
+- Collected metrics are written to AWS [AMP workspace](https://aws.amazon.com/prometheus/)
+
+#### 2. Set DCL network chain id in ansible inventory
+[`deployment/ansible/inventory/aws/group_vars/all.yaml`]
+```yaml
+chain_id: test-net
+...
+```
+<details>
+<summary>Example for Testnet 2.0</summary>
+
+```yaml
+chain_id: testnet-2.0
+...
+```
+</details>
+
+## Deployment
+#### 1. Run terraform
+```bash
+cd deployment/terraform/aws
+terraform apply
+```
+
+> **_Note:_** Terraform asks a confirmation before applying changes
+
+#### 2. Generate ansible inventory from terraform output
+Once terraform completes successfully, run the following command to genarate ansible inventory file:
+```bash
+terraform output -json ansible_inventory | dasel -r json -w yaml . > ../../ansible/inventory/aws/aws_all.yaml
+```
+
+#### 3. (Optional) Consider enabling state sync
+When joining an existing pool, you may want to enable state sync for all the nodes.<br>
+To do so, you should set state sync parameters:
+```yaml
+config:
+...
+  statesync:
+    enable: true
+    rpc_servers: "http(s):<node1-IP>:26657,..."
+    trust_height: <trust-height>
+    trust_hash: "<trust-hash>"
+...
+```
+in the following ansible config files
+```text
+deployment/ansible/roles/configure/vars/
+  validator.yml
+  private-sentry.yml
+  observer.yml
+  public-senrty.yml
+  seed.yml
+```
+
+<details>
+<summary>Example for Testnet 2.0</summary>
+
+```yaml
+config:
+  statesync:
+    enable: true
+    rpc_servers: "https://on.test-net.dcl.csa-iot.org:26657,https://on.test-net.dcl.csa-iot.org:26657"
+    
+```
+</details>
+
+> **_NOTE:_**  You should provide at least 2 addresses for `rpc_servers`. It can be 2 identical addresses
+
+You can use the following command to obtain `<trust-height>` and `<trust-hash>` of your network
+
+```bash
+curl -s http(s)://<host>:<port>/commit | jq "{height: .result.signed_header.header.height, hash: .result.signed_header.commit.block_id.hash}"
+```
+<details>
+<summary>Example for Testnet 2.0</summary>
+
+```bash
+curl -s https://on.test-net.dcl.csa-iot.org:26657/commit | jq "{height: .result.signed_header.header.height, hash: .result.signed_header.commit.block_id.hash}"
+```
+</details>
+
+- `<host>` - RPC endpoint host of the network being joined
+- `<port>` - RPC endpoint port of the network being joined
+
```

Comment:
What about setup of persistent peers? How it can be done? Please also provide a note about persistent peers for the Test Net.